plugins/source/aws/resources/services/elasticsearch/tags_fetch.go

```diff
@@ -0,0 +1,35 @@
+package elasticsearch
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/service/elasticsearchservice"
+	"github.com/aws/aws-sdk-go-v2/service/elasticsearchservice/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func resolveTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
Does this fix anything? Is it worth adding this to the changelog too?

Reply:
No, it is for other es resources